src/Mutator/Util/MutatorsGenerator.php

```diff
@@ -99,7 +99,7 @@ private function registerFromClass(string $mutator, $setting)
                 $this->mutatorList[$mutator] = [];
             }
         } elseif (!array_key_exists($mutator, $this->mutatorList) || $this->mutatorList[$mutator] === []) {
-            $this->mutatorList[$mutator] = (array) $setting;
```

Comment:
we support `true`, `false` and an `array`. First two are handled in the code above, that's why this type casting is not needed

Reply:
Hands down type conversion here is beyond MutatorsGenerator responsibilities. No wonder you wanted to remove i. At some point it should be refactored out of here, but probably not right now.